package.json

```diff
@@ -28,7 +29,7 @@
   },
   "devDependencies": {
     "moment": "2.14.1",
```

Comment:
Actually, I think it was because I wanted to use new APIs in garbanzo like `isSameOrAfter`. Moment has had a lot of updates since 2.9.0, both features and bugfixes that we should be taking advantage of.